### DISCUSSION TITLE
SQL Query Logging

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4575] [Feature Requests]
Directus converts actions into SQL queries, it would be nice to store a log of all these raw queries that are executed.

**Comment by rijkvanzanten** [comment]
Set `LOG_LEVEL` to trace 👍🏻  ✨ 

If you want to save these to disk, use `LOG_STYLE="raw"` and something like https://github.com/pinojs/pino-tee to save it to file with `npx directus start | pino-tee trace ./my-logs` 👍🏻
